api/db/migrate/20171217130805_create_phone_numbers.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+class CreatePhoneNumbers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :phone_numbers do |t|
+      t.string :user_id
+      t.string :number
```

Comment:
Add not null constraints for user_id and number columns

Reply:
Why is user_id string? All ids in rails are going to be ints added by default. 
Also avoid adding foreign keys as simple fields. 
Use rail's `add_reference` or `add_foreign_key`, this may be useful: https://stackoverflow.com/questions/22815009/add-a-reference-column-migration-in-rails-4